### ISSUE TITLE
Behavior when tweet contains both image and YouTube URL

### ISSUE THREAD

**Issue by brw** [FxEmbed/FxEmbed#576]
When a tweet contains both an image and a YouTube URL, FixTweet embeds the YouTube video instead of the image. Twitter embeds the image only, and so does BetterTwitFix. I would expect FixTweet to follow the same behavior as Twitter, but I could also see this different behavior being intentional. I'd be curious to know if that's the case here.

Example: https://fxtwitter.com/0SenEi_/status/1738912299136713091
Screenshot of the differences in behavior: 
![image](https://github.com/FixTweet/FixTweet/assets/7560938/ee23e13c-ec93-4e99-9825-517086dbed56)

In this example tweet I wanted to highlight the art, but it embedded the video instead. This tweet also happens to be a quote tweet, but I don't think that matters here. I see the same behavior with this different tweet I found: https://fxtwitter.com/Underevent2020/status/1739026933906551096

**Comment by dangeredwolf** [comment]
Yeah, we recently (https://github.com/FixTweet/FixTweet/commit/7d759f8dc7ae5d86aec70f23c9bf8421ca3467e8) made a change which ensures that YouTube embeds in particular still show up as long as a YouTube link is included.

I made this change from seeing that people often include images in YouTube video announcement tweets. Notably tweets like this which up until that change would break the video player because they also included an image.
![image](https://github.com/FixTweet/FixTweet/assets/2636265/69f51945-f587-4825-a223-3d4cf57bb9eb)

I was going to see what happens if you force the photo in the URL, but I actually unintentionally uncovered a different bug. This definitely does not look right at all! It's in a weird state where it partially thinks there is a video attached but there isn't one, causing the text of the tweet to become duplicated.
![image](https://github.com/FixTweet/FixTweet/assets/2636265/6e35bffd-3bf6-47ad-85bb-f6cde2fd8d15)

I also think the new gallery view should also ignore external media if there is already media in the tweet.

![image](https://github.com/FixTweet/FixTweet/assets/2636265/7649b847-8fef-4522-bd8f-42d91e84ee93)

As for tweets like the one in question I'm not 100% sure what the best way to control this behavior is, because I can see a lot of scenarios where this both makes sense and doesn't make sense (like this one).

**Comment by brw** [comment]
Yea, I totally get defaulting to the YouTube embed, as that's probably the more useful embed in most cases. I've also noticed the trend of attaching thumbnails to tweets. Not sure why people do it though, maybe to force them to go to the YouTube website? Or maybe it's an algorithm thing. But either way it breaks the YouTube embed on the Twitter website as well.

Using the photo URL is a good workaround, I hadn't thought of that. Also didn't know about the gallery view, that could work pretty well too. But then if a tweet has multiple images and a YouTube link you still can't get both the text and all of the images together (unless there's a way to link to multiple photos at once that I don't know of), which might be a bit of a shame. Perhaps it would be nice to expose this scenario somehow to still have the option to match default Twitter behavior as well (discarding the YouTube embed).

Regarding what the plain fxtwitter links show by default, that's up to you I suppose. I also see the pros and cons of either approach there.

**Comment by dangeredwolf** [comment]
Hmmm.... we use i.fxtwitter.com / i.fixupx.com right now to force Instant View on Telegram but I'm starting to think this makes far more sense to use to force images in general instead of videos. We use instant view on photos anyway to make them expandable on mobile so in most cases this could actually serve two functions at once actually o.O 

Having a force image subdomain can help in scenarios like these:
- This one, where a YouTube video is laying on top of an image you want to be able to expand
- Current behavior is that Twitter videos are prioritized over images in a tweet. Which usually makes sense, as that has traditionally been what people used fxtwitter for before Discord Twitter embeds broke 100%, but in some cases you might want to show the image or the mosaic of the other images instead. Currently you can view just one photo by adding `/photo/2` for example, but not show *all* the rest. Plus, adding `i.` is a lot easier than ad ding /photo/1 at the end anyway.

**Comment by dangeredwolf** [comment]
I forgot to update the issue to say that images are prioritized in general now anyway over YouTube embeds
